examples/03_connectivity/plot_seed_based_connectome.py

```diff
@@ -1,57 +1,89 @@
 """
-Extract signals and plot a connectome for seed-based atlases
-============================================================
+Extract signals and plot a connectome for atlas-based spheres
```

Comment:
And don't hesitate to rename the example: "plot_sphere_based_connectome", I would say.